src/Persistent/Api.hs

```diff
@@ -353,18 +329,18 @@ instance FromProto Delegation where
     fromProto (ProtoDelegation ctx f t) m = Delegation m ctx f t
 
 -- | So far `mkMetaInfo` is only used by `nextMetaInfo`.
-mkMetaInfo :: User -> Timestamp -> AUID a -> MetaInfo a
-mkMetaInfo user now oid = MetaInfo
+mkMetaInfo :: User -> AUID User -> Timestamp -> AUID a -> MetaInfo a
+mkMetaInfo cUser uid now oid = MetaInfo
```

Comment:
Yes, you are right, I have that contract and failed to document it. I separated the argument to make sure `addFirstUser` doesn't diverge, while bootstrapping. In this form it's rather obvious it doesn't, but I'm sure there are ways of expressing that, which don't require the extra argument (and the contract).